external-crates/move/solana/move-to-solana/src/stackless/translate.rs

```diff
@@ -128,7 +128,8 @@ impl<'up> GlobalContext<'up> {
         #[cfg(feature = "solana")]
         assert!(account_address::AccountAddress::ZERO.len() == 32);
 
-        debug!(target: "globalenv", "{:#?}", env);
+        // Note: printing global env genarates huge output, but you can do it with the following line
+        // debug!(target: "globalenv", "{:#?}", env);
```

Comment:
This comment points to the exact place where printing this info is the most useful (but really bulky).
I do not want to investigate next time I debug compiler where would be the best place for this line. So that is why I left this comment.

Reply:
If you look at your next comment you'll see that this `debug!` is operated by "nodes", but it will be printed for target "debug" as well. And I need target debug while debugging...
But any way, I do not fight for this minor issue, this PR does not worth it. I'll just keep this code unchanged.